Makefile

```diff
@@ -151,6 +163,10 @@ config-generated.go:
 
 $(TARGET): $(SOURCES) $(GENERATED_FILES) Makefile | show-summary
 	$(QUIET_BUILD)go build -i -o $@ .
+	$(QUIET_BUILD)go build -i -ldflags "-X main.commit=${COMMIT} -X main.version=${VERSION} -X main.libExecDir=${LIBEXECDIR}" -o $@ .
+
+pause: $(PAUSEDIR)/$(PAUSEBIN).o
```

Comment:
Hi @gorozco1, 

We can install `pause` from the default rule, and after this is merged, we can remove `.ci/install_virtcontainers.sh` from `clearcontainers/tests` repo.